src/main/java/org/spongepowered/common/data/key/KeyRegistry.java

```diff
@@ -112,6 +112,7 @@ public static void registerKeys() {
         keyMap.put("item_enchantments", makeListKey(ItemEnchantment.class, of("ItemEnchantments")));
         keyMap.put("banner_patterns", makeListKey(PatternLayer.class, of("BannerPatterns")));
         keyMap.put("banner_base_color", makeListKey(DyeColor.class, of("BannerBaseColor")));
+        keyMap.put("elder", makeSingleKey(Boolean.TYPE, Value.class, of("Elder")));
```

Comment:
Oh, my mistake - assuming it was the name of the DataQuery, whoops.

Only just got an instance going so couldn't test - builds were failing because I hadn't updated the DamageSource Access Transformers.